### DISCUSSION TITLE
Where can I find the project file of ripes so I can open it with Qt?

### DISCUSSION THREAD

**Discussion by Y1annis** [mortbopet/Ripes#229] [General]
Hello, I am working on ripes and I am trying to open the project using Qt but I cant find the project file. 

Thank you for your time.

**Comment by Y1annis** [comment]
Basically my question is How can I change some of the src code and recompile into a new executable?

**Comment by mortbopet** [comment]
Ripes is CMake based - see https://doc-snapshots.qt.io/qtcreator-4.0/creator-project-cmake.html and uses the following top-level project file: https://github.com/mortbopet/Ripes/blob/master/CMakeLists.txt

**Reply by Y1annis** [reply]
Do I need Qt5? Because I am using Qt6 and I am getting an error on these find_package(Qt5Core REQUIRED)
find_package(Qt5Widgets CONFIG REQUIRED)
find_package(Qt5Svg REQUIRED)
find_package(Qt5Charts CONFIG REQUIRED) that these files cannot be find

**Reply by mortbopet** [reply]
Yep, Ripes is currently Qt5 based - it should probably be moved to Qt6 soon :) (the issue is mainly wrt. backwards compatibility - afaik Qt6 doesn't support ubuntu 18.04 (though it might be time to stop supporting that...).

**Reply by Y1annis** [reply]
Ok so I need to download what I need in Qt5, the ubuntu version that I am using is 20.04 I think

**Reply by Y1annis** [reply]
When I try to compile it using Qt5 now I get an error that Ripes/external/VSRTL and libelfin doesnt have a CMakelist.txt file
and that a command line option -mbig -obj is unrecognised.

And thank you a lot for all the help

**Reply by mortbopet** [reply]
You most likely have not pulled the submodules of the project.

```sh
git clone --recursive https://github.com/mortbopet/Ripes.git
```

**Reply by Y1annis** [reply]
I tried getting the project from Git and I am getting these 2 errors still.

![image](https://user-images.githubusercontent.com/109152409/191756730-cc9336d2-b012-4371-a969-d3864133dbe0.png)
